spec/models/commit_monitor_branch_spec.rb

```diff
@@ -71,6 +72,20 @@
     end
   end
 
+  context ".github_pr_uri" do
```

Comment:
Sounds good.  Also I think I'll change the name of the first test to something that makes more sense as this method doesn't actually take user and repo as parameters.